apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerException.cs

```diff
@@ -84,6 +86,25 @@ public bool IsRetryable
             }
         }
 
+        /// <summary>
+        /// For retryable <see cref="SpannerException"/>, this value represents
+        /// the recommended minimum retry delay. It might or might not
+        /// be present, even if this exception is retryable.
+        /// </summary>
+        public TimeSpan? RecommendedRetryDelay { get; }
+
+        /// <summary>
+        /// For retrying aborted commits, we need to diferentiate whether the operation
+        /// was aborted due to a session expiry. If the session expired we cannot
+        /// retry automatically because we cannot reuse the session.
+        /// Note that it is us who are setting the <see cref="ErrorCode"/> to <see cref="ErrorCode.Aborted"/>
```

Comment:
I think this part would be better as a non-XML comment, just in case this property is ever made public.